src/store/authors/thunk.js

```diff
@@ -0,0 +1,15 @@
+import { addAuthor, getAuthors } from '../../services';
+import { store_addAuthor, store_getAuthors } from './actionCreators';
+
+export const thunk_getAuthors = () => async (dispatch) => {
+	await getAuthors().then((response) =>
+		dispatch(store_getAuthors(response.data.result))
+	);
+};
+
+export const thunk_addAuthor = (author, token) => async (dispatch) => {
+	await addAuthor(author, token).then(() => {
+		dispatch(store_addAuthor(author));
```

Comment:
you should dispatch getAuthors info, not the store update, once you get it, the store will be updated itself

Reply:
Yeah, it's an artifact from previous HW, where such changes were only store-wide. Got rid of a pretty amount of lines in the code.